ui/app/pages/send/send-content/send-gas-row/send-gas-row.component.js

```diff
@@ -35,7 +35,11 @@ export default class SendGasRow extends Component {
 
   renderAdvancedOptionsButton () {
     const { metricsEvent } = this.context
-    const { showCustomizeGasModal } = this.props
+    const { showCustomizeGasModal, advancedInlineGasShown } = this.props
+    // Tests should behave in same way as mainnet, but are using Localhost
+    if (advancedInlineGasShown === true && !process.env.IN_TEST) {
```

Comment:
nit: it'd be more consistent with everything to skip the `=== true`
```suggestion
    if (advancedInlineGasShown && !process.env.IN_TEST) {
```